sdk/storage/storage-file-share/review/storage-file-share.api.md

```diff
@@ -167,6 +179,7 @@ export interface DirectoryForceCloseHandlesHeaders {
     errorCode?: string;
     marker?: string;
     numberOfHandlesClosed?: number;
+    numberOfHandlesFailedToClose?: number;
```

Comment:
unfortunately we already have inconsistency between File and Directory...

Reply:
wait File and Directory are consistent, but why did we have `closedHandlesCount` on `CloseHandlesInfo`

Reply:
> wait File and Directory are consistent, but why did we have `closedHandlesCount` on `CloseHandlesInfo`

Our convenience layer did some wrapping when sum up the num of all the handles closed when calling `forceCloseAllHandles`. @XiaoningLiu may have more context.
SInce it's already shipped, we would better align with it.